specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,402 @@
+          "format": "int64"
+        },
+        "communicationsManagerContactId": {
+          "type": "integer",
+          "description": "Communication Manager's contact id for this incident.",
+          "format": "int64"
+        },
+        "siteReliabilityContactId": {
+          "type": "integer",
+          "description": "Site Reliability Manager's contact id for this incident.",
+          "format": "int64"
+        },
+        "diagnosticsLink": {
+          "type": "string",
+          "description": "Diagnostics link for this incident."
+        },
+        "isOutage": {
+          "type": "boolean",
+          "description": "A value indicating whether incident is an outage or not."
+        },
+        "occuringLocation": {
+          "type": "object",
+          "description": "Location of the entity that raised the incident.",
+          "properties": {
+            "environment": {
+              "type": "string",
+              "description": "Environment of the incident."
+            },
+            "datacenter": {
+              "type": "string",
+              "description": "Datacenter name of the incident."
+            },
+            "role": {
+              "type": "string",
+              "description": "Role name of the incident."
+            },
+            "instance": {
+              "type": "string",
+              "description": "Instance or cluster name of the incident."
+            },
+            "component": {
+              "type": "string",
+              "description": "Component name of the incident."
+            },
+            "slice": {
+              "type": "string",
+              "description": "service-specific id of the slice."
+            }
+          }
+        },
+        "bridges": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/incidentBridge"
+          },
+          "description": "A list of incident bridges."
+        },
+        "owningTenantName": {
+          "type": "string",
+          "description": "Owning service for this incident."
+        },
+        "owningTeamName": {
+          "type": "string",
+          "description": "Owning team for this incident."
+        },
+        "contactAlias": {
+          "type": "string",
+          "description": "Contact alias for the incident."
+        },
+        "createdBy": {
+          "type": "string",
+          "description": "Someone who created the incident."
+        },
+        "impactStartTime": {
+          "type": "string",
+          "description": "Impact start time for the incident."
+        },
+        "acknowledgeBy": {
+          "type": "integer",
+          "description": "Acknowledgement id for the incident.",
+          "format": "int64"
+        },
+        "incidentManagerAlias": {
+          "type": "string",
+          "description": "Incident manager alias for the incident."
+        },
+        "acknowledgeTime": {
```

Comment:
>acknowledgeTime [](start = 9, length = 15)

format: date-time?

Reply:
same below.

---
In reply to: [552323004](https://github.com/Azure/azure-rest-api-specs/pull/12206#discussion_r552323004) [](ancestors = 552323004)

Reply:
addressed